templates/todo/projects/csharp-sql-swa-func/.repo/bicep/infra/main.bicep

```diff
@@ -24,6 +24,11 @@ param storageAccountName string = ''
 param sqlDatabaseName string = ''
 param sqlServerName string = ''
 param webServiceName string = ''
+param apimServiceName string = ''
+
+@description('Flag to use Azure API Management to mediate the calls between the Web frontend and the backend API')
+@allowed([ 'true', 'false', '' ])
+param useAPIM string
```

Comment:
@jongio I know in the past we've had bugs with non string parameters, but we should be in a better place now.  I think long term we want to change this to:

```bicep
@description("When true, use Azure API Management to mediate the calls between the Web frontend and the backend API.")
param useAPIM bool = false
```

Reply:
If we are releasing the CLI updates with fixes for param types, then we could do it now.

Could you respond here with a comment on which build of the CLI  this should work with?

Reply:
It's in `daily`, so we should be able to uptake this. Making a change here.